robottelo/constants/__init__.py

```diff
@@ -1725,17 +1725,17 @@ class Colored(Box):
 STRING_TYPES = ['alpha', 'numeric', 'alphanumeric', 'latin1', 'utf8', 'cjk', 'html']
 
 VMWARE_CONSTANTS = {
-    'cluster': 'Satellite-Engineering',
+    'cluster': 'SatQE-Cluster',
```

Comment:
We already have this defined in settings, so I don't think we should hardcode it here, so would suggest updating tests to use settings instead of this

Reply:
Updated.